packages/components/datetime/src/types.ts

```diff
@@ -1,3 +1,8 @@
 export type DateType = Date | string | number;
 
-export type DateFormat = 'full' | 'time' | 'weekday' | 'day';
+export type DateFormat =
+  | 'full-with-seconds'
```

Comment:
```suggestion
  | 'fullWithSeconds'
```

For values of props we prefer to use camelCase